src/type-system.ts

```diff
@@ -354,6 +361,33 @@ export const ElysiaType = {
 			})
 			.Encode((value) => value) as any as TNumber
 	},
+	Integer: (property?: NumberOptions): TInteger => {
```

Comment:
`IntegerOptions` doesn't exist?

Reply:
applied in 3173e52281ce11025b3567d0a74ecf0298636a2c